examples/save_disp.py

```diff
@@ -0,0 +1,68 @@
+import os
+from pathlib import Path
+from packaging import version
+
+import torch
+import numpy as np
+from PIL import Image
+
+
+def load_image(imfile):
+    """Utility function to load image files as tensors"""
+    img = Image.open(imfile)
+    img = np.array(img).astype(np.uint8)
+    img = torch.from_numpy(img).permute(2, 0, 1).float()
+    img = img.to("cuda")
+
+    assert img.ndim == 3, f"Expected 3D tensor, got shape {img.shape}"
+    return img
+
+
+def main():
+    # Turn off deprecated nvfuser if Torch version is old.
+    # Displays multiple warnings otherwise.
+    if version.parse(torch.__version__) < version.parse("2.2.0"):
+        torch._C._jit_set_nvfuser_enabled(False)
+
+    with torch.no_grad():
+        model_path = Path("output") / "monster-mix-script.pt"
+        assert os.path.exists(model_path), f"Model file not found at {model_path}"
+
+        model = torch.jit.load(model_path, map_location="cuda")
+        model.eval()
+
+        print("TorchScript model loaded")
+
+        image_left_path = Path("input") / "left" / "sample_l.png"
+        image_right_path = Path("input") / "right" / "sample_r.png"
+
+        assert os.path.exists(image_left_path), (
+            f"Left image not found at {image_left_path}"
+        )
+        assert os.path.exists(image_right_path), (
+            f"Right image not found at {image_right_path}"
+        )
+
+        image_left = load_image(image_left_path)
+        image_right = load_image(image_right_path)
+
+        print("Stereo image pair loaded")
+
+        disparity = model(image_left, image_right)
+        torch.cuda.synchronize()
```

Comment:
Why synchronise?

Reply:
Thanks! I'll remove it. Explicit synchronization is not necessary in this case because PyTorch handles it for us in the expressions following this line of code; in `disparity = disparity.cpu().numpy()`.

Reply:
Yeah, we generally only want to do that when debugging stuff, fyi.